namerd/iface/interpreter-thrift/src/main/scala/io/buoyant/namerd/iface/ObserverCache.scala

```diff
@@ -32,11 +34,14 @@ class MaximumObservationsReached(maxObservations: Int)
  *                       instead.
  * @param inactiveCapacity The maximum size of the inactive cache.  LRU eviction is used to
  *                         maintain this constraint.
+ * @param inactiveTTLSecs The amount of time, in seconds, to keep observer in inactive cache before
+ *                        expiring them.
  * @param mkObserver The function to use to create new Observers if they are not in either cache.
  */
 class ObserverCache[K <: AnyRef, T](
   activeCapacity: Int,
   inactiveCapacity: Int,
+  inactiveTTLSecs: Long,
```

Comment:
This interface would be a little bit nicer if it took a `com.twitter.util.Duration` here instead of a Long or Int.

Reply:
I thought about that but I wanted to be consistent with the rest of the config - all time-related parameters are defined as a numeric value ex. "retryJitterSecs" or "ttlSecs" or "decayTimeMs"